src/vs/workbench/contrib/extensions/browser/extensionEditor.ts

```diff
@@ -843,7 +850,7 @@ export class ExtensionEditor extends BaseEditor {
 		const contributes = manifest.contributes;
 		const colors = contributes && contributes.colors;
 
-		if (!colors || !colors.length) {
```

Comment:
These changes also don't seem like they are part of this fix. I'm going to merge this since they don't look too bad but in the future please split out any refactorings from the rest of the change

Reply:
OK.